src/main/java/it/pagopa/interop/probing/scheduler/annotations/validator/StringArrayValidator.java

```diff
@@ -0,0 +1,30 @@
+
+package it.pagopa.interop.probing.scheduler.annotations.validator;
+
+import it.pagopa.interop.probing.scheduler.annotations.ValidateStringArraySize;
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class StringArrayValidator implements ConstraintValidator<ValidateStringArraySize, String[]> {
+
+	private int maxSize;
+
+	@Override
+	public void initialize(ValidateStringArraySize constraintAnnotation) {
+		maxSize = constraintAnnotation.maxSize();
+	}
+
+	@Override
+	public boolean isValid(String[] array, ConstraintValidatorContext context) {
+		if (Objects.nonNull(array)) {
+			for (String s : array) {
```

Comment:
You could use lambda functions ex: Filter and FindFirst

Reply:
It's ok for me, since this was taken from registry-updater project and should be then standardized, i propose to create a technical debt task for this implementation to take in charge in the next sprint. Would it be ok for you?

Reply:
For me it is good but this is old fashion java programming. However it is your decision.

Reply:
it's ok we'll apply this standardization with the technical debt task in the next sprint, i'll create it and add it to the backlog